### PR TITLE
Refactor "guide view" and apply new design (+ some qol changes)

### PR DESCRIPTION
**Pull request by Roxas92**
Disclaimer: I know I should make smaller PRs 🙈 but while I was working on the guide view I came across with some things I definitely wanted to change right now - so pls don't kill me for that. :D

___

To give some context to some of the commits

## Role View (4d5b219d5c92aa5814073990e847742fdaf13d30) -> (#51)
- Added a new css class `.markdown` which can be used to style blank tags created from markdown. This is a "living" class so we definitely have to make additions to here once we know all of our needs (especially from the content creators). I also added some sample text to scholar in order to test it.
- Re: [layouts/partials/job/widgets/guides_list.html](https://github.com/The-Balance-FFXIV/glam/blob/4d5b219d5c92aa5814073990e847742fdaf13d30/layouts/partials/job/widgets/guides_list.html) - This is very much hardcoded right now, I don't know if it makes sense to refactor this in the future once the structure of alles jobs are clear

## Ressources Partial (ef63f1d38c83defdeaf817aa825217c2c4e76f50)
- It totally make sense to have two ressources partials, one for "job" and one for "role", so we can easily display them on different pages.

## Custom Tailwind Utility Class: "Card" (34c04b928ceab9d43a925871d4aaca4a5d4fa9c6)
- While I was working on the Role View, I noticed that even with having several card partials, they often use the same code as well as we sometimes need to use the bare card wrapper without any additions. Therefore we now can use `.card` to have the basic card styling and in addition to this `.card-[color]` to define the border color. So the prefered usage will be `class="card card-healer"`. If you want to only have the card (padding, background, but without border) you can just use `class="card"`

## Fix Slider Styling (ed02fceb63415806b4ab7c1610ca3b9180ea1b17) -> (#70)
- The slider (class `tabbed`) can now be used regardless the context. Previously some preapplied styling caused weird behaviour.
- Made some small adjustments to fit into the design

### CODE REVIEW

**Review by nonowazu — CHANGES_REQUESTED** [review]
2 very tiny changes and some questions.

File: exampleSite/content/jobs/healers/scholar/basic-guide.md [hunk 1]
```diff
@@ -16,13 +16,21 @@ changelog:
     message: This article didn't exist
```
[the remark this answers is not in the document]
Comment (Roxas92) [reply exampleSite/content/jobs/healers/scholar/basic-guide.md:16]: In this specific case I added the content manually in the code editor (everything below the `---`), for other things like the teaser images from the other PR I used the admin editor to add / change things.

File: layouts/jobs/single.html [hunk 2]
```diff
@@ -1,105 +1,34 @@
...
+  </div>
 
-<div class="mx-20 my-20">
+  <div class="container mx-auto">
     {{ partial "components/guide_meta_slider.html" . }}
-</div>
+  </div>
+
+  <div class="py-14 bg-no-repeat bg-cover" style="background-image: url('/theme-assets/jobs/{{ lower .Parent.Title }}_resources_background.png');">
+    {{ partial "job/job_ressources.html" .Parent }}
```
[the remark this answers is not in the document]
Comment (Roxas92) [reply layouts/jobs/single.html:28]: Oh well ... In germany we have the same word with the same meaning but we write it with two `ss`. I might have made that wrong in all of my previous projects as well. Thank you for finding that. :D

File: layouts/jobs/single.html [hunk 3]
```diff
@@ -1,105 +1,34 @@
...
+  </div>
+
+  {{ partial "job/role_ressources.html" .Parent }}
```
[the remark this answers is not in the document]
Comment (Roxas92) [reply layouts/jobs/single.html:31]: These are two different partials, one for the `role` and one for the `job`.

File: layouts/partials/cards/default.html [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-	<div class="bg-card-light px-5 py-6 border-l-8 border-{{ $borderClass }}">
+	<div class="card card-{{ $borderClass }} space-y-3">
     {{ if isset . "name" }}
     <div class="border-b border-card-dark pb-8">
       {{ partial "cards/_header.html" . }}
     </div>
     {{ end }}
-    {{ .content }}
+    <div>
+      {{ .content }}
```
[the remark this answers is not in the document]
Comment (Roxas92) [reply layouts/partials/cards/default.html:16]: This is not meant to be the global variable `.Content`. When calling the partial the `.Content` variable gets mapped to `content`. e.g. https://github.com/Roxas92/glam/blob/4d5b219d5c92aa5814073990e847742fdaf13d30/layouts/partials/job/fundamentals.html#L7,  https://github.com/Roxas92/glam/blob/4d5b219d5c92aa5814073990e847742fdaf13d30/layouts/partials/job/fundamentals.html#L95-L103

**Comment by Roxas92** [comment]
In the last commit I also addressed some shadow and font issues I noticed while I compared the result with the mockup.

File: layouts/jobs/single.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (nonowazu) [reply layouts/jobs/single.html]: I see that now; apologies. It was late when I reviewed last night.

File: layouts/partials/cards/default.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (nonowazu) [reply layouts/partials/cards/default.html:16]: In this case, I find it a bit misleading. I personally think we should rename this dictionary key to something else, or name the key `Content` so it's clear that it's still the content data from whatever called the partial.

**Comment by nonowazu** [comment]
Would it be possible to not expand PRs mid-review in the future? There's now 4 PRs to juggle and my short-term memory can only hold so much information :(

File: exampleSite/content/jobs/healers/scholar/basic-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nonowazu) [reply exampleSite/content/jobs/healers/scholar/basic-guide.md:16]: Alright. I'm just trying to make sure that we're using the editor as much as possible to make sure that the example interface(s) we're making are relevant and actually function.

**Comment by Roxas92** [comment]
Of course, apologies for that. I force pushed the version before and will create a new PR for the design adjustments once this PR is done.

File: layouts/partials/cards/default.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (Roxas92) [reply layouts/partials/cards/default.html]: Alright, changed to to `text` instead.

**Review by nonowazu — APPROVED** [review]

**Pull request merged**